samples/server/petstore/php-slim4/composer.json

```diff
@@ -1,4 +1,6 @@
 {
+  "name": "GIT_USER_ID/GIT_REPO_ID",
+  "version": "1.0.0",
```

Comment:
Generally, the version should NOT be added to the composer.json, it's deduced by Packagist from Git tags (namely, tags become the version).